flow/envs/multiagent/base.py

```diff
@@ -322,7 +322,3 @@ def apply_rl_actions(self, rl_actions=None):
         # clip according to the action space requirements
         clipped_actions = self.clip_actions(rl_actions)
         self._apply_rl_actions(clipped_actions)
-
-    def set_iteration_num(self):
-        """Increment the number of training iterations."""
-        self.num_training_iters += 1
```

Comment:
kind of need this function for curricula. I suppose I can just move it into the subclasses

Reply:
this was moved to flow/envs/base.py. It can be accessed by multiagent environments as well.